chain-evm/src/state/account.rs

```diff
@@ -15,11 +15,51 @@ pub struct Account {
     /// Account data storage.
     pub storage: Storage,
     /// EVM bytecode of this account.
-    pub code: Box<[u8]>,
+    pub code: Vec<u8>,
```

Comment:
`Box` had some benefits: a smaller struct size (which could get significant if the ledger has a lot of value accounts), and a speed bump against any accidental attempts to modify the code in place, which should never be needed. Just like `Vec<u8>`, `Box<[u8]>` provides all slice methods through deref.

Reply:
I mainly changed it because the evm crate uses Vec<u8>, and it seemed simpler to avoid having to make Vecs out of Boxes whenever data has to go through the evm code. I'll look into this.

Reply:
`Box<[u8]>` is restored in the latest commit.